### ISSUE TITLE
Required Preprocessing?

### ISSUE THREAD

**Issue by vkumar1997** [Open-Debin/Emotion-FAN#12]
Could you please elaborate on the pre-processing used for the AFEW dataset? I am currently training on the aligned faces originally provided and only getting 40% val precision (after 60 epochs. It slows considerably after that). I am assuming that there is something wrong with pre-processing.

Also, could you provide the weights of the final trained model?

**Comment by Open-Debin** [comment]
Faces detection and alignment are implemented by Dlib. My processing detail is shown in my paper. Also, I recommend you to use the pre-train model, Resnet18_FER+_pytorch.pth.tar that I have shared. 

Would you be kind enough to assign me a star?

**Comment by vkumar1997** [comment]
Thank you for your response. I am already doing the pre-processing but only reaching 45% on AFEW (It maybe because I am not giving 25% extra padding in the facial bounding boxes).

**Comment by Open-Debin** [comment]
@vkumar1997 
Merry Christmas, I recently update the Emotion-FAN, new features include data process, environment install, CK+ code, Baseline code, and more detail instructions. Also, you can find the old version directory of Emotion-FAN in the README.md. I hope my new updates can help you greatly. Please see the Emotion-FAN for more details.
